### DISCUSSION TITLE
generating of the thumbnails and image display

### DISCUSSION THREAD

**Discussion by Gordolin** [hbstack/.github#40] [Q&A] [answered]
Hi, i think, the thumbnails will be generated somehow from the middle of an image. So sometimes the generated thumbnails look not very nice to me. Is  it possible to adjust the rendering start point lets say to left top corner?

The module for displaying images has no posibilty to zoom in an image over its size. In your hbs theme this is possible. I liked this very much especially on small images. Is it possible to implement these feature in your new theme?

**Comment by razonyang** [comment, marked as answer]
I remember that the theme resized the images instead of cropping it, but the image will take full width and fixed height (for keeping cards styling equal height and look well), so there will probably a part won't be viewable. The HBS theme layout is different, since the image take fixed height and auto width that preserve aspect ratio, which means you can view the full picture. Because of the difference of the layout, I don't see this will be changed in the new theme, unless we have an ideal solution.

But you can change the object position other than center by [adding custom CSS](https://hbstack.dev/docs/advanced/style/).

```scss
.hb-blog-post-card-img {
    object-position: top;
}
```

**Reply by Gordolin** [reply]
Thanks, that works fine for me
